dag_in_context/src/greedy_dag_extractor.rs

```diff
@@ -857,15 +857,15 @@ impl CostModel for DefaultCostModel {
             "Base" | "TupleT" | "TNil" | "TCons" => 0.,
             "Int" | "Bool" | "Float" => 0.,
             // Algebra
-            "Add" | "PtrAdd" | "Sub" | "And" | "Or" | "Not" => 10.,
-            "FAdd" | "FSub" => 50.,
+            "Add" | "PtrAdd" | "Sub" | "And" | "Or" | "Not" | "Shl" | "Shr" => 10.,
+            "FAdd" | "FSub" | "Fmax" | "Fmin" => 50.,
```

Comment:
If you have any justification for these costs it's worth a comment. Otherwise, it's fine we can play with these